src/sass/themes/dark/components/_panels.scss

```diff
@@ -0,0 +1,10 @@
+body.theme-dark {
+  .scrolling-panel {
+    margin-right: 5px;
+    margin-top: 5px;
```

Comment:
what's the intent here? it looks like the `margin-right` leaves a small gap between the highlighted channel and the right edge:

<img width="250" alt="screen shot 2018-08-01 at 1 36 48 pm" src="https://user-images.githubusercontent.com/820696/43538416-fd346126-958f-11e8-8146-31b80784ce2d.png">

Reply:
i fixed this on master